Based on the latest IR 2030 update [1], update this range to cover
5725-5850 MHz rather than 5725-5875 MHz, increase the max EIRP to
200 mW, remove the DFS requirement, and add NO-OUTDOOR.

[1] https://www.ofcom.org.uk/__data/assets/pdf_file/0028/84970/ir-2030.pdf

Signed-off-by: Seth Forshee <seth.forshee@canonical.com>

diff --git a/db.txt b/db.txt
index ddf9e14..a5829e0 100644
--- a/db.txt
+++ b/db.txt
@@ -599,7 +599,7 @@ country GB: DFS-ETSI
 	(5250 - 5350 @ 80), (100 mW), NO-OUTDOOR, DFS, AUTO-BW, wmmrule=ETSI
 	(5470 - 5730 @ 160), (500 mW), DFS, wmmrule=ETSI
 	# short range devices (ETSI EN 300 440-1)
-	(5725 - 5875 @ 80), (100 mW), DFS
+	(5725 - 5850 @ 80), (200 mW), NO-OUTDOOR
 	# 60 GHz band channels 1-6
 	(57000 - 71000 @ 2160), (40)